docs/Reference/API-Methods.md

```diff
@@ -2590,16 +2642,18 @@ Polls the specified filter and returns an array of changes that have occurred si
     }
 
     ```
-    
-    
+
+
 
 ### eth_getFilterLogs
 
 Returns an array of [logs](../Concepts/Events-and-Logs.md) for the specified filter.
 
+Improve log retrieval performance by using [`admin_generateLogBloomCache`](admin_generatelogbloomcache).
```

Comment:
Suggest rewording to make active: 

Use [`admin_generateLogBloomCache`](admin_generatelogbloomcache) to improve log retrieval performance.